osu.Game/Rulesets/Difficulty/Skills/Skill.cs

```diff
@@ -58,6 +61,21 @@ public void Process(DifficultyHitObject current)
             Previous.Push(current);
         }
 
+        public void TouchProcess(DifficultyHitObject current)
```

Comment:
This code affects all difficulty calculators and should not be in here.

Reply:
Issue has been resolved as a result of reverting strain calculation change completely.